components/youtube_data_api_custom_app/actions/update-video-details/update-video-details.mjs

```diff
@@ -0,0 +1,74 @@
+import youtubeDataApi from "../../youtube_data_api_custom_app.app.mjs";
+import common from "../../../youtube_data_api/actions/update-video-details/common.mjs";
+
+export default {
+  ...common,
+  key: "youtube_data_api_custom_app-update-video-details",
+  name: "Update Video Details",
+  description: "Updates a video's metadata. [See the docs](https://developers.google.com/youtube/v3/docs/videos/update) for more information",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    youtubeDataApi,
+    videoId: {
+      propDefinition: [
+        youtubeDataApi,
+        "userOwnedVideo",
+      ],
+    },
+    title: {
+      propDefinition: [
+        youtubeDataApi,
+        "title",
+      ],
+    },
+    description: {
+      propDefinition: [
+        youtubeDataApi,
+        "description",
+      ],
+      optional: true,
+    },
+    privacyStatus: {
```

Comment:
I noticed the props in this file and in `youtube_data_api_custom_app` are different